spikeextractors/extractors/yassextractors/yassextractors.py

```diff
@@ -0,0 +1,63 @@
+from spikeextractors import SortingExtractor
+from spikeextractors.extractors.numpyextractors import NumpyRecordingExtractor
+import numpy as np
+from pathlib import Path
+from spikeextractors.extraction_tools import check_valid_unit_id
+
+try:
+    import h5py
+    HAVE_SCSX = True
+except ImportError:
+    HAVE_SCSX = False
```

Comment:
```suggestion
HAVE_YASS = True
```

As only numpy is needed, no need to check for installation